File: AdoptPet_Server/AdoptPet.Domain/Entities/DonorPetAudit.cs
using AdoptPet.Domain.Common.Interfaces;

namespace AdoptPet.Domain.Entities
{
    public class DonorPetAudit : IEntity<int>
    {
        public int Id { get; set; }
        public DateTime LastDonation { get; set; }
        public int Version { get; set; }
        public decimal NewTotalDonation { get; set; }
        public decimal OldTotalDonation { get; set; }
        public DateTime CreatedAt { get; set; }
        public bool IsDeleted { get; set; }

        // khoa ngoai
        public DonorPet DonorPet { get; set; }
    }
}

File: AdoptPet_Client/src/app/shared/models/donor/DonorPetAudit.ts
import { PetDto } from "../pet/PetDto";
import { DonorDto } from "./DonorDto";

export interface DonorPetAudit {
    id: number;
    lastDonation: Date
   version: number;
   newTotalDonation: number;
   oldTotalDonation: number;
   isDeleted: boolean;
   donor: DonorDto
   pet: PetDto
}
